main/acle.md

```diff
@@ -360,8 +360,10 @@ Armv8.4-A [[ARMARMv84]](#ARMARMv84). Support is added for the Dot Product intrin
   to use keyword attributes instead of GNU-style attributes.
 * Added missing word to Function Multi Versioning's [Name mangling](#name-mangling).
 * Added description of SVE reinterpret intrinsics.
-* Changes for [Function Multi Versioning](#function-multi-versioning):
+* Changes and fixes for [Function Multi Versioning](#function-multi-versioning):
   * Added [MOPS](#memcpy-family-of-operations-intrinsics---mops).
+  * Align priorities to account for feature dependencies.
+  * Introduce alternative names (aliases) `rdma` for `rdm`.
```

Comment:
Align -> aligned
Introduce -> Introduced

Keep verb form consistent with previous in list.